pkg/apis/ibmcloud/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -20,6 +20,7 @@ limitations under the License.
 package v1alpha1
 
 import (
+	v1 "k8s.io/api/core/v1"
```

Comment:
@xunpan for this auto-generated code, I prefer we commit this in another commit for future PR.

Reply:
ok.